README.md

```diff
@@ -39,10 +45,91 @@ point in having those in this fork for various reasons.
+| ZLIB_COMPAT              | --zlib-compat            | Compile with zlib compatible API                                                             | OFF                              |
+| ZLIB_ENABLE_TESTS        |                          | Build test binaries                                                                          | ON                               |
+| WITH_GZFILEOP            | --with-gzfileops         | Compile with support for gzFile related functions                                            | OFF                              |
+| WITH_MSAN                | --with-msan              | Build with memory sanitizer                                                                  | OFF                              |
+| WITH_OPTIM               | --without-optimizations  | Build with optimisations                                                                     | ON                               |
+| WITH_NEW_STRATEGIES      | --without-new-strategies | Use new strategies                                                                           | ON                               |
+| WITH_NATIVE_INSTRUCTIONS |                          | Instruct the compiler to use the full instruction set on this host (gcc/clang -march=native) | OFF                              |
+|                          | --force-sse2             | Assume SSE2 instructions are always available                                                | DISABLED (x86), ENABLED (x86_64) |
+| WITH_ACLE                | --without-acle           | Build with ACLE CRC                                                                          | ON                               |
+| WITH_NEON                | --without-neon           | Build with NEON intrinsics                                                                   | ON                               |
+| WITH_DFLTCC_DEFLATE      | --with-dfltcc-deflate    | Use DEFLATE COMPRESSION CALL instruction for compression on IBM Z                            | OFF                              |
+| WITH_DFLTCC_INFLATE      | --with-dfltcc-inflate    | Use DEFLATE COMPRESSION CALL instruction for decompression on IBM Z                          | OFF                              |
+| WITH_SANITIZERS          | --with-sanitizers        | Build with address sanitizer and all supported sanitizers other than memory sanitizer        | OFF                              |
+| WITH_FUZZERS             | --with-fuzzers           | Build test/fuzz                                                                              | OFF                              |
+
+Install
+-------
+
+We do not recommend installing unless you know what you are doing, as this can
+override the system default zlib library, and any wrong configuration or
+incompatibility of zlib-ng can make the whole system unusable.
+
+On linux distros, an alternative way to use zlib-ng instead of zlib, is through
```

Comment:
"Linux" should be written with capitalized L as it's trademark, not generic noun.